eth/scripts/deploy_stable_swap.js

```diff
@@ -1,9 +1,22 @@
-const { ethers } = require("hardhat");
+const { ethers, upgrades, network } = require("hardhat");
```

Comment:
I tried to run this script for this branch, and this gave me below error
```
make devnet-eth
[...]
marcin@marcin-Latitude-3520 ~/g/m/eth (swap-in-most)> npx hardhat run scripts/deploy_stable_swap.js                                  (base) 
bazero deployed to: 0x5FbDB2315678afecb367f032d93F642f64180aa3
LP deployed to: 0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512
Stable Swap deployed to: 0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0
Deploying MostL2...
TypeError: Cannot read properties of undefined (reading 'guardianIds')
    at main (/home/marcin/git/most/eth/scripts/deploy_stable_swap.js:50:14)
```

Reply:
Even though I have `L2_KEY` env set locally.

Reply:
You need to run it like so, see --network:
```
L2_KEY=xxx npx hardhat run --network ldwa ./scripts/deploy_stable_swap.js 

```